src/apps/loyalty/__tests__/server.tsx

```diff
@@ -0,0 +1,21 @@
+import { Home, Login } from "../server/route_handlers"
+
+describe ("Home page", () => {
+  it("redirects to /inquiries", () => {
+    let req = { baseUrl: "loyalty" } as any
+    let res = { redirect: jest.fn() } as any
+    let next = jest.fn() as any
+    Home(req, res, next)
+    expect(res.redirect).toHaveBeenCalledWith("loyalty/inquiries")
+  })
+})
+
+describe ("Login page", () => {
+  it("renders the login page", () => {
+    let req = { baseUrl: "loyalty", csrfToken: () => "token" } as any
+    let res = { redirect: jest.fn(), send: jest.fn() } as any
+    let next = jest.fn() as any
+    Login(req, res, next)
+    expect(res.send).toHaveBeenCalledWith("test")
```

Comment:
This is where I want to assert a couple things about the string (which is the rendered DOM and CSS/JS), namely, i'll just be making sure some of the form html exists (and is pointing to the right place).